### ISSUE TITLE
Request for PyCharm Plugin Support

### ISSUE THREAD

**Issue by songyuc** [JudiniLabs/code-gpt-docs#213]
Hello CodeGPT Team,

I am really interested in using your CodeGPT extension and think it immensely helpful for my coding workflow. I was wondering if there are any plans to develop a plugin for PyCharm users?

As PyCharm is a popular IDE among Python developers, having CodeGPT support for it would be fantastic and would certainly benefit the community.

Thank you for considering this request, and for the great work on the CodeGPT extension.

Best regards,
Yucheng Song

**Comment by rgmedidata** [comment]
I agree with this request. It would help lot of tester community to use gpt features in coding.

**Comment by PilarHidalgo** [comment]
We apologize for the bad experience you had. We invite you to download the new Jetbrain extension here: https://codegpt.co/jetbrains-plugin
